tools/browser-matcher/common-matchers.mjs

```diff
@@ -110,3 +110,38 @@ export const supportsBFCache = new SpecMatcher()
   .include('firefox')
   .include('ios')
   // .include('android>9.0') -- does not work on android 9.0 emulator (v100 Chrome) for unknown precise reason;
+
+export const supportsFirstPaint = new SpecMatcher()
+  .include('chrome>=60')
+  .include('edge>=79')
+  .include('android>9.0')
+
+export const supportsFirstContentfulPaint = new SpecMatcher()
+  .include('chrome>=60')
+  .include('edge>=79')
+  .include('android>9.0')
+  .include('firefox>=84')
+  .include('safari>15') // this should be >= 14.1 but safari 15 on Sauce hates FCP, and it destroys the other tests on the same thread too...
```

Comment:
Has this been re-reviewed with the move to wdio? If it still an issue, maybe we should get a ticket in to re-review this after the move to LT.

Reply:
Probably doesn't matter anymore since the oldest Safari we test is 16 at this point, but i'll throw a ticket on the backlog to re-look into this